app/helpers/application_helper/toolbar/hosts_center.rb

```diff
@@ -188,7 +190,9 @@ class ApplicationHelper::Toolbar::HostsCenter < ApplicationHelper::Toolbar::Basi
           :url_parms => "main_div",
           :confirm   => N_("Shutdown the selected items?"),
           :enabled   => false,
-          :onwhen    => "1+"),
+          :onwhen    => "1+",
+          :klass   => ApplicationHelper::Button::HostFeatureButtonWithDisable,
+          :options => {:feature => :shutdown}),
         button(
```

Comment:
@rwsu You don't want to add the button class that is checking `@record` instance variable, into **this** toolbar.

This toolbar is for list of Hosts, and because you don't have one exact selected, the `@record` variable is `nil`. The toolbar buttons here in this toolbar are enabled by JavaScript. *(that's what the `:onwhen` attribute is there)*.

The `host_center.rb` is probably the right one, not `hosts_center.rb` ;-)

*(I'm sorry, I know it's a bit complicated :smile:)*